linera-views/tests/operations_tests.rs

```diff
@@ -205,103 +208,180 @@ async fn test_readings_memory_specific() {
     run_readings_vec(key_value_store, key_value_vec).await;
 }
 
-#[cfg(test)]
-async fn run_writings_random<OP: KeyValueStoreClient + Sync>(key_value_store: OP) {
-    let mut rng = rand::rngs::StdRng::seed_from_u64(2);
-    let mut kv_state = BTreeMap::new();
-    let n_oper = 100;
-    let size_batch = 8;
-    // key space has size 4^4 = 256 so we necessarily encounter collisions
-    // because the number of generated keys is about size_batch * n_oper = 800 > 256.
-    let key_prefix = vec![0];
-    for _ in 0..n_oper {
+fn generate_random_batch<R: RngCore>(
+    rng: &mut R,
+    key_prefix: &[u8],
+    size_batch: usize,
```

Comment:
```suggestion
    batch_size: usize,
```

(Or just `size`; it's clear from the context.)

Reply:
Ok, changed.